### ISSUE TITLE
Pictures is still being processed after 2 days

### ISSUE THREAD

**Issue by beer119** [mapillary/mapillary_issues#980]
Hi
1 week ago I did do my first upload to mapillary and back then it took less than 12 hours before the pictures was processed. 2 days ago I did upload a bunch of new pictures but now they are still in the "still processin".

is this normal?

**Comment by peterneubauer** [comment]
No, this is not normal. We are importing a large batch from Victoria, Australia, which broke different parts of the system being too stressed. We are currently working through it, should be good in a couple of days and catching up. Sorry for the inconvenience - we are scaling the system at the same time.

**Comment by beer119** [comment]
Good luck and good processing

**Comment by peterneubauer** [comment]
Thanks @beer119 , we are on it - it's almost a million pictures :)

**Comment by ireun** [comment]
16 days and still queued for processing.. :D How many photos left from Australia? ; >

**Comment by beer119** [comment]
There was no problem with procceing pictures yesterday but those pictures that I have been uploading today has not started with the procceing yet

**Comment by ireun** [comment]
Well.. To be clear I'm talking about this processing... Images are already on a map.

![2015-05-19_18-25-56](https://cloud.githubusercontent.com/assets/695818/7708220/954487f6-fe54-11e4-8569-d3a48a3b3021.png)

**Comment by peterneubauer** [comment]
This is because the last processing wave us actually not run I them yet. We
are going to decrease that lag shortly.

**Comment by richlv** [comment]
i suspect this issue has been resolved and could be closed ? :)

**Comment by beer119** [comment]
It has been resolved for me at least. I cannot talk for the others
